.travis.yml

```diff
@@ -1,16 +1,18 @@
 language: python
 python:
-    - "2.7"
+    - "3.5"
     - "3.6"
+    - "3.7"
+    - "3.8"
 env:
     - TORNADO_VERSION=5.0
     - TORNADO_VERSION=4.5.3
     - TORNADO_VERSION=3.2.2
```

Comment:
Can you update the version here to just be 6.0? Since this change is dropping support for anything prior to 6.0, we can remove the test runs for all versions below 6 as well.

Reply:
Please make the updates in both .travis.yml as well as tox.ini